2025-06-02 15:45:45 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 15:45:45 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 15:45:45 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 15:45:45 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 15:45:45 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 15:47:11 [WARNING] aiogram.dispatcher: Received SIGINT signal
2025-06-02 15:53:01 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 15:53:01 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 15:53:01 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 15:53:01 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 15:53:01 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 15:54:49 [WARNING] aiogram.dispatcher: Received SIGINT signal
2025-06-02 15:54:52 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 15:54:52 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 15:54:52 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 15:54:52 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 15:54:52 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 15:54:57 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 15:55:00 [ERROR] aiogram.event: Cause exception while process update id=471135836 by bot id=7727141953
NameError: name 'incident_types_kb' is not defined
Traceback (most recent call last):
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/handlers/tickets.py", line 39, in process_incident
    reply_markup=incident_types_kb()
                 ^^^^^^^^^^^^^^^^^
NameError: name 'incident_types_kb' is not defined
2025-06-02 15:55:02 [DEBUG] bot.handlers.tickets: Call cancel_creation
2025-06-02 15:56:34 [WARNING] aiogram.dispatcher: Received SIGINT signal
2025-06-02 15:56:37 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 15:56:37 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 15:56:37 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 15:56:37 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 15:56:37 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 15:56:46 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 15:56:48 [DEBUG] bot.handlers.tickets: Call process_incident
2025-06-02 15:56:48 [ERROR] aiogram.event: Cause exception while process update id=471135844 by bot id=7727141953
NameError: name 'incident_types_kb' is not defined
Traceback (most recent call last):
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/handlers/tickets.py", line 40, in process_incident
    reply_markup=incident_types_kb()
                 ^^^^^^^^^^^^^^^^^
NameError: name 'incident_types_kb' is not defined
2025-06-02 15:57:31 [WARNING] aiogram.dispatcher: Received SIGINT signal
2025-06-02 15:57:33 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 15:57:33 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 15:57:33 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 15:57:33 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 15:57:33 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 15:57:38 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 15:57:39 [DEBUG] bot.handlers.tickets: Call process_incident
2025-06-02 15:58:27 [WARNING] aiogram.dispatcher: Received SIGINT signal
2025-06-02 16:00:55 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 16:00:55 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 16:00:55 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 16:00:55 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 16:00:56 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 16:01:07 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 16:01:08 [DEBUG] bot.handlers.tickets: Call process_incident
2025-06-02 16:01:17 [DEBUG] bot.handlers.tickets: Call cancel_creation
2025-06-02 16:06:31 [DEBUG] bot.handlers.tickets: Call cancel_creation
2025-06-02 16:06:38 [DEBUG] bot.handlers.tickets: Call cancel_creation
2025-06-02 16:16:54 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 16:16:55 [DEBUG] bot.handlers.tickets: Call process_request
2025-06-02 16:51:58 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 16:51:58 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 16:51:58 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 16:51:58 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 16:51:58 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 16:52:05 [ERROR] aiogram.event: Cause exception while process update id=471135862 by bot id=7727141953
TypeError: select_type() got an unexpected keyword argument 'dispatcher'
Traceback (most recent call last):
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/middlewares/navigation.py", line 33, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/handlers/funcs.py", line 8, in wrapper
    result = await func(message, state, *args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: select_type() got an unexpected keyword argument 'dispatcher'
2025-06-02 16:52:10 [ERROR] aiogram.event: Cause exception while process update id=471135864 by bot id=7727141953
TypeError: select_type() got an unexpected keyword argument 'dispatcher'
Traceback (most recent call last):
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/middlewares/navigation.py", line 33, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/handlers/funcs.py", line 8, in wrapper
    result = await func(message, state, *args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: select_type() got an unexpected keyword argument 'dispatcher'
2025-06-02 16:53:51 [WARNING] aiogram.dispatcher: Received SIGINT signal
2025-06-02 16:53:54 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 16:53:54 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 16:53:54 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 16:53:54 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 16:53:54 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 16:54:03 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 16:54:04 [ERROR] aiogram.event: Cause exception while process update id=471135868 by bot id=7727141953
TypeError: process_incident() got an unexpected keyword argument 'dispatcher'
Traceback (most recent call last):
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/middlewares/navigation.py", line 33, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/handlers/funcs.py", line 8, in wrapper
    result = await func(message, state, *args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: process_incident() got an unexpected keyword argument 'dispatcher'
2025-06-02 16:54:05 [ERROR] aiogram.event: Cause exception while process update id=471135869 by bot id=7727141953
TypeError: process_request() got an unexpected keyword argument 'dispatcher'
Traceback (most recent call last):
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/middlewares/navigation.py", line 33, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/handlers/funcs.py", line 8, in wrapper
    result = await func(message, state, *args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: process_request() got an unexpected keyword argument 'dispatcher'
2025-06-02 16:54:06 [ERROR] aiogram.event: Cause exception while process update id=471135870 by bot id=7727141953
TypeError: cancel_creation() got an unexpected keyword argument 'dispatcher'
Traceback (most recent call last):
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/middlewares/navigation.py", line 33, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/.cache/pypoetry/virtualenvs/bot-YUzCg1qZ-py3.12/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/dyuzhev_mn/Python/bot/src/bot/handlers/funcs.py", line 8, in wrapper
    result = await func(message, state, *args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cancel_creation() got an unexpected keyword argument 'dispatcher'
2025-06-02 16:55:28 [WARNING] aiogram.dispatcher: Received SIGINT signal
2025-06-02 16:55:33 [INFO] bot.config_handlers.logging_config: Logging configured successfully
2025-06-02 16:55:33 [DEBUG] bot.config_handlers.logging_config: Log file initialized: logs/bot.log
2025-06-02 16:55:33 [DEBUG] asyncio: Using selector: EpollSelector
2025-06-02 16:55:33 [INFO] bot.config_handlers.logging_config: Starting bot
2025-06-02 16:55:33 [INFO] bot.config_handlers.logging_config: Меню для бота загруженно
2025-06-02 16:55:40 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 16:55:41 [DEBUG] bot.handlers.tickets: Call cancel_creation
2025-06-02 16:55:46 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 16:55:47 [DEBUG] bot.handlers.tickets: Call process_incident
2025-06-02 16:55:53 [DEBUG] bot.handlers.tickets: Call cancel_creation
2025-06-02 16:55:57 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 16:55:59 [DEBUG] bot.handlers.tickets: Call process_request
2025-06-02 16:56:07 [DEBUG] bot.handlers.tickets: Call cancel_creation
2025-06-02 16:56:10 [DEBUG] bot.handlers.tickets: Call select_type
2025-06-02 16:56:17 [DEBUG] bot.handlers.tickets: Call process_incident
